blockchain/beacon/src/drand.rs

```diff
@@ -208,8 +232,18 @@ impl DrandBeacon {
 
         if cfg!(debug_assertions) && config.network_type == DrandNetwork::Mainnet {
             let server = config.server;
+            let tokio_config = config.tokio.clone();
             let remote_chain_info = std::thread::spawn(move || {
-                let rt = tokio::runtime::Runtime::new()?;
+                let rt = tokio::runtime::Builder::new_multi_thread()
```

Comment:
Maybe we should keep new constructor here? What is the motive for creating a second tokio runtime here?

Reply:
Indeed, it's redundant. Reverted